test/tests/pbs_admin_suspend.py

```diff
@@ -405,3 +413,416 @@ def test_degraded_resv(self):
         # See reservation in degreaded state
         a = {'reserve_state': (MATCH_RE, 'RESV_DEGRADED|10')}
         d = self.server.expect(RESV, a, rid)
+
+    @timeout(400)
+    def test_resv_jobend(self):
+        """
+        Test if a node goes back to free state when reservation ends and
+        admin-suspended job is killed
+        """
+
+        # Submit a reservation
+        r = Reservation(TEST_USER)
+        r.set_attributes({'Resource_List.select': '1:ncpus=1',
+                          'reserve_start': time.time() + 60,
+                          'reserve_end': time.time() + 120})
+        rid = self.server.submit(r)
+
+        # See reservation is confirmed
+        a = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+        d = self.server.expect(RESV, a, rid)
```

Comment:
No it does take single value. id=rid would be good if there are multiple values in a list